### PR TITLE
chore: upgrade to opentelemetry 0.22.0

### PR DESCRIPTION
**Pull request by seemk**
Upgrade to OpenTelemetry 0.22 which itself depends on `@opentelemetry/api` `1.0.0`

### CODE REVIEW

File: packages/instrumentation-express/test/opentelemetry-express-layers.spec.ts [hunk 1]
```diff
@@ -672,8 +672,8 @@ describe('opentelemetry-express-layers', () => {
         describe('multiple routes', () => {
             it('two sibling routes, first is calling res.end()', async () => {
                 app.get('/:firstRouteParam', resEndMiddleware);
-                app.get('/:secondRouteParam', (req, res, next) => {
-                    throw Error('should not be invoked');
+                app.get('/:secondRouteParam', (_req: express.Request, _res: express.Response) => {
```
Comment (YanivD) [inline packages/instrumentation-express/test/opentelemetry-express-layers.spec.ts:675]: @seemk Thanks!!!

**Comment by blumamir** [comment]
Thanks @seemk ,
Can you please also upgrade the dev dependency on `opentelemetry-instrumentation-mocha` to `0.0.1-rc.3`?
I published it as well with core packages `^0.22.0` and api `^1.0.0`

**Review by blumamir — APPROVED** [review]

**Comment by seemk** [comment]
> Thanks @seemk ,
> Can you please also upgrade the dev dependency on `opentelemetry-instrumentation-mocha` to `0.0.1-rc.3`?
> I published it as well with core packages `^0.22.0` and api `^1.0.0`

Thanks, added it

**Comment by blumamir** [comment]
v0.22.0 published

**Pull request merged**